src/pspm_sf.m

```diff
@@ -225,7 +229,26 @@
         return;
       end
     end
-    events = data{1}.data;
+    switch class(options.marker_chan_num_event)
```

Comment:
as in other functions, I suggest removing this option entirely and just always use the *first* marker channel.